code/drasil-example/dblpend/lib/Drasil/DblPend/Assumptions.hs

```diff
@@ -1,34 +1,30 @@
 {-# LANGUAGE PostfixOperators #-}
 module Drasil.DblPend.Assumptions (twoDMotion, cartSys, cartSysR,
-  yAxisDir, startOriginSingle, startOriginDouble, firstPend, secondPend, assumpSingle, assumpDouble) where
+  yAxisDir, startOriginDouble, firstPend, secondPend, assumpBasic, assumpDouble) where
     
 import Language.Drasil
 import Language.Drasil.Chunk.Concept.NamedCombinators
 import qualified Language.Drasil.Sentence.Combinators as S
 
 import Data.Drasil.Concepts.Documentation (assumpDom) 
 import Data.Drasil.Concepts.Math (cartesian, xAxis, yAxis, direction, origin, positive)
-import Data.Drasil.Concepts.Physics (gravity, twoD, pendulum)
+import Data.Drasil.Concepts.Physics (gravity, twoD)
 import Drasil.DblPend.Concepts (pendMotion, firstRod, secondRod, firstObject, secondObject)
 
 assumpBasic :: [ConceptInstance]
 assumpBasic = [twoDMotion, cartSys, cartSysR, yAxisDir]
 
-assumpSingle :: [ConceptInstance]
-assumpSingle = assumpBasic ++ [startOriginSingle]
-
 assumpDouble :: [ConceptInstance]
-assumpDouble = assumpSingle ++ [startOriginDouble, firstPend, secondPend]
+assumpDouble = assumpBasic ++ [startOriginDouble, firstPend, secondPend]
 
-twoDMotion, cartSys, cartSysR, yAxisDir, startOriginSingle, startOriginDouble,
+twoDMotion, cartSys, cartSysR, yAxisDir, startOriginDouble,
   firstPend, secondPend:: ConceptInstance 
 
 twoDMotion        = cic "twoDMotion"    twoDMotionDesc          "twoDMotion"    assumpDom
 cartSys           = cic "cartSys"       cartSysDesc             "cartSys"       assumpDom
 cartSysR          = cic "cartSysR"      cartSysRDesc            "cartSysR"      assumpDom
 yAxisDir          = cic "yAxisDir"      yAxisDirDesc            "yAxisDir"      assumpDom
-startOriginSingle = cic "startOrigin"   startOriginDescSingle   "startOrigin"   assumpDom
-startOriginDouble = cic "startOrigin"   startOriginDescDouble   "startOrigin"   assumpDom
+startOriginDouble = cic "startOrigin2x" startOriginDescDouble   "startOrigin"   assumpDom
```

Comment:
This change ("startOrigin" to "startOrigin2x") creates a lot of noise in the output. Which is sad, as this means that UIDs are used for way way too much. But let's go with that for now, as at least it's clear what happened.